src/OpenDiffix.Core.Tests/Anonymizer.Tests.fs

```diff
@@ -77,6 +83,25 @@ let ``anon count(col)`` () =
   |> evaluateAggregator diffixCount [ aidColumnList; strColumn ]
   |> should equal (Integer 30L)
 
+[<Fact>]
+let ``anon sum(real col)`` () =
+  // - 1 user with Null real is ignored
```

Comment:
The list markers don't add much value imo.

Reply:
not sure what you mean. The `-` characters themselves, the contents or...?

Anyhow, I tried to follow the convention from this file.

Reply:
> The - characters themselves, the contents or...?

Yes, the `-` followed by `//` seem redundant.